assets/app/views/_deployment-config-metadata.html

```diff
@@ -1,11 +1,7 @@
 <div ng-if="deploymentConfigId != ''" class="metadata">
   <span>Created from deployment config {{deploymentConfigId}}</span>
-  <span class="pficon-layered" ng-if="!exists" data-toggle="tooltip" data-placement="right" title="Deployment config no longer exists" style="cursor: help;">
-    <span class="pficon pficon-warning-triangle"></span>
-    <span class="pficon pficon-warning-exclamation"></span>
+  <span ng-if="!exists" data-toggle="tooltip" data-placement="right" title="Deployment config no longer exists" class="pficon pficon-error-circle-o" style="cursor: help;">
   </span>
```

Comment:
go ahead move this closing span up to the same line since it has no children now